interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -144,217 +142,214 @@
+                                        <td class='currentvalues p-2'><input type="text" class="form-control" size='5'
+                                            name='BMI' id='BMI_input' value="{if $vitals->get_BMI() != 0}{$vitals->get_BMI()|substr:0:5|attr}{/if}"/></td>
+                                        {foreach item=result from=$results}
+                                            <td class='historicalvalues'>{if $result.BMI != 0}{$result.BMI|substr:0:5|text}{/if}</td>
+                                        {/foreach}
+                                    </tr>
+
+                                    <tr>
+                                        <td>{xlt t="BMI Status"}</td><td>{xlt t="Type"}</td>
+                                        <td class='currentvalues p-2'><input type="text" class="form-control" size='20'
+                                        name="BMI_status" id="BMI_status' value="{$vitals->get_BMI_status()|attr}"/></td>
+                                        {foreach item=result from=$results}
+                                        <td  class='historicalvalues'>{if $result.BMI_status}{xlt t=$result.BMI_status}{/if}</td>
+                                        {/foreach}
+                                    </tr>
+
+                                    <tr>
+                                        <td>{xlt t="Other Notes"}</td>
+                                        <td>&nbsp;</td>
+                                        <td class='currentvalues p-2'><input type="text" class="form-control" size='20'
+                                        name="note" id='note' value="{$vitals->get_note()|attr}" /></td>
+                                        {foreach item=result from=$results}
+                                            <td class='historicalvalues'>{$result.note|text}</td>
+                                        {/foreach}
+                                    </tr>
+                                    <tr><td>&nbsp;</td></tr>
+
+                                    <tr>
+                                        <td colspan='5' class="text-center">
+                                            {if $patient_age <= 20 || (preg_match('/month/', $patient_age))}
+                                            <!-- only show growth-chart button for patients < 20 years old -->
+                                            <input type="button" class="btn btn-primary" id="pdfchart" value='{xla t="Growth-Chart"} ({xla t="PDF"})'>
+                                            <input type="button" class="btn btn-primary" id="htmlchart" value='{xla t="Growth-Chart"} ({xla t="HTML"})'>
+                                            {/if}
+                                        </td>
+                                    </tr>
+                            </tbody>
+                        </table>
+                    </div>
 
                 <div class="form-group clearfix">
```

Comment:
couple lines below, looks like lost 2 lines of updated in merge that should look like:
```
-                            <button type="submit" class="btn btn-primary btn-save editonly" name="Submit" value=''>{xlt t="Save"}</button>
-                            <button type="button" class="btn btn-secondary btn-cancel editonly" id="cancel" value=''>{xlt t="Cancel"}</button>
```